packages/altive_lints/example/lints/avoid_single_child.dart

```diff
@@ -70,14 +70,47 @@ class MyWidget extends StatelessWidget {
             ],
           ],
         ),
+        // expect_lint: avoid_single_child
+        Column(
+          children: [
+            if(isValued)...[
+              Container(),
+            ]else...[
+              Container(),
+            ],
+          ],
+        ),
+        // expect_lint: avoid_single_child
+        Column(
+          children: [
+            if(isValued)...[
+              Container(),
+            ],
+          ],
+        ),
+        Column(
+          children: [
+            if(isValued)...[
+              Container(),
+              Container(),
+            ],
+          ],
+        ),
+        Column(
+          children: [
+            if(isValued)...[
+              Container(),
+            ],
+            Container(),
+          ],
+        ),
         Column(
           children: random.nextBool()
               ? [
                   const Text('Hello World'),
                 ]
               : [
                   const Text('Hello'),
-                  const Text('World'),
                 ],
```

Comment:
![badge](https://img.shields.io/badge/review-note-yellowgreen)
This is not the subject of this Issue, but this pattern has one element each, so lint is the expected value.